### DISCUSSION TITLE
Multiple select inputs on form

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#2680] [Q&A] [answered]
Hey Team, 
We are implementing a for that has multiple choice dropdown and I wanted to know if the option are long, Can we shorten the letters on the badges or, do we have to leave them complete. (see examples below)

**SHORT VERSION**

<img width="394" alt="Screen Shot 2022-09-14 at 2 01 48 PM (1)" src="https://user-images.githubusercontent.com/72419222/190498520-37629e40-6660-4298-b7e5-bdb8f703b7cf.png">

**WORST-CASE COMPLETE VERSION**

<img width="833" alt="Screen Shot 2022-09-14 at 2 11 02 PM" src="https://user-images.githubusercontent.com/72419222/190498523-2b381395-b53a-4bd5-ba65-c86dfd88457f.png">

**MID-POINT CASE COMPLETE VERSION**

<img width="830" alt="Screen Shot 2022-09-14 at 2 11 33 PM" src="https://user-images.githubusercontent.com/72419222/190498533-14bdf71a-1f35-41d4-9278-13b90217cdc2.png">

**Comment by loreina** [comment, marked as answer]
Hi @lgalloherran,

We have a new Multi-select Combobox that's available in Figma and will be available in code in the next week or so. With this design, the selection will show up as pills within the input field. They span to the width of the field and will wrap to the next line if the pill needs more space. 

<img width="441" alt="image" src="https://user-images.githubusercontent.com/23108291/190511437-bdd76190-1143-4c47-8944-a28ab7ff8fd3.png">

You could also use our [Truncate component](https://paste.twilio.design/components/truncate) so the pills would be truncated at a certain character length. On hover, the entire string of text will appear. We would caution against truncating text though as it limits readability of the selection and visually hides information on a page where a user might have to spend more time to be able to see the full text/selection.

Let us know if you have any questions! Thanks!
